apps/not-arrived/translations/en/default.json

```diff
@@ -1,1038 +1,358 @@
-{
```

Comment:
perhaps we should just delete the default.json file? I added hof-transpile as part of the postinstall in another PR so we can assume this is always built?